Update icons size and fixed over scroll

tshlos opened the pull request:
Hey Diego, just did some minor changes on icon properties and wrote comments on the lines I changed.

Deohgu reviewed (changes requested):
### Game.jsx :
The margin right was working as intended.
_marginRight_ is the camel cased version of margin-right.
You can read about it [here](https://www.pluralsight.com/guides/react-inline-styling).

I think that issue might just be connected with the over-scrolling. So maybe fixing that first with solve this one.

---

### Game.styled.js :
The over scrolling issue seems to remain, the width or height don't go over 100vw/vh so that shouldn't be a problem. The 80vw amount is mostly a stylistic a choice. 
So something else must be causing extra space that pushes over the page size.
Maybe try removing the margins of GameStyled, and then it can be re-center using Flex on the parent component instead.

tshlos commented:
Hi Diogo, do you mind sharing a screenshot of what you see so I can have a
better idea? Thanks

On Tue, Jan 12, 2021 at 4:03 AM Diogo Santos <notifications@github.com>
wrote:

> *@Deohgu* requested changes on this pull request.
> Game.jsx :
>
> The margin right was working as intended.
> *marginRight* is the camel cased version of margin-right.
> You can read about it here
> <https://www.pluralsight.com/guides/react-inline-styling>.
>
> I think that issue might just be connected with the over-scrolling. So
> maybe fixing that first with solve this one.
> ------------------------------
> Game.styled.js :
>
> The over scrolling issue seems to remain, the width or height don't go
> over 100vw/vh so that shouldn't be a problem. The 80vw amount is mostly a
> stylistic a choice.
> So something else must be causing extra space that pushes over the page
> size.
> Maybe try removing the margins of GameStyled, and then it can be re-center
> using Flex on the parent component instead.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Deohgu/minesweeper/pull/21#pullrequestreview-566202858>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AOLA3DRPWHUS7ZKQP37MOELSZQ3C3ANCNFSM4V6B4Q3A>
> .
>

Deohgu commented:
Of course.
Sorry, maybe over scroll wasn't the best term. So when the window width decreases to 800px, the height of the app becomes larger than the window and overflows so you get the vertical scroll bar. I'm assuming that it might be due to the extra margins.
The intend behaviour is the app would stay centred without overflowing.

<img src="https://user-images.githubusercontent.com/45941521/104357161-bc298580-5504-11eb-9c19-aba18e51b3d2.png" width="600">

tshlos commented:
I see.. is it ok if I still work on this? Thanks for sharing the screenshot.

Deohgu commented:
Yes that's perfectly fine.
Have a look at my initial review of your Pull-Request for ideas.
When you're finished just request another review.

tshlos commented:
Will do, thanks!

On Tue, Jan 12, 2021 at 11:13 AM Diogo Santos <notifications@github.com>
wrote:

> Yes that's perfectly fine.
> Have a look at my initial review of your Pull-Request for ideas.
> When you're finished just request another review.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Deohgu/minesweeper/pull/21#issuecomment-758875368>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AOLA3DS5NGGLHCY36MKUYADSZSNMDANCNFSM4V6B4Q3A>
> .
>

Deohgu reviewed (changes requested):
I've commented on the specific parts of the code.

tshlos commented on src/components/Game/Game.styled.js:
```diff
@@ -1,33 +1,35 @@
+  right: 0;
+  bottom: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
```
I see what you mean. Thanks for the suggestion.

tshlos commented:
Hey Diego, I just updated with the changes you suggested. Let me know what
you think.

On Wed, Jan 13, 2021 at 12:01 PM Diogo Santos <notifications@github.com>
wrote:

> *@Deohgu* requested changes on this pull request.
>
> I've commented on the specific parts of the code.
> ------------------------------
>
> In src/components/Game/Game.styled.js
> <https://github.com/Deohgu/minesweeper/pull/21#discussion_r556766489>:
>
> > +  width: 100%;
> +  height: 100%;
>
> Here we lose the ratio of the app.
>
> I recommend something like this instead :
>
>   // You keep these
>   max-width: calc(800px * 0.8);
>   max-height: calc(800px * 0.92);
>
>   // But add something like this
>   // When the size of the page decreases lower than the apps max-width
>   // It maintains the width at 100 and the height is the width * 15% to maintain the ratio.
>   @media (max-width: calc(800px * 0.8)) {
>     width: 100vw;
>     height: calc(100vw * 1.15);
>   }
>
> ------------------------------
>
> In src/components/Game/Game.styled.js
> <https://github.com/Deohgu/minesweeper/pull/21#discussion_r556773274>:
>
> >    @media (min-width: 800px) {
> -    font-size: 63.68px;
> +    font-size: 55px;
> +  }
> +  @media (max-width: 500px) {
> +    font-size: 25px;
>
> The media query for 500px doesn't quite work here because at *width <
> 500px* it jumps to font-size: 25px.
>
> So we could keep the original font-size: calc(8vw), but change at line 28 @media
> (min-width: 800px) { to update at the right size of @media (min-width:
> calc(800px * 0.8)).
> ------------------------------
>
> In src/components/Game/Game.styled.js
> <https://github.com/Deohgu/minesweeper/pull/21#discussion_r556774612>:
>
> > +  position: fixed;
> +  display: block;
> +  margin: auto;
> +  left: 0;
> +  right: 0;
> +  bottom: 0;
> +  top: 0;
>
> Great, it is working. Another option would also be to add
> I also checked it out and no you don't need display: block so that can be
> removed.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Deohgu/minesweeper/pull/21#pullrequestreview-567532603>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AOLA3DS5A2WTAMAVEAW3QGTSZX33NANCNFSM4V6B4Q3A>
> .
>

Deohgu reviewed (approved):
Looks great, good job!
Ready to merge with the master.

tshlos commented:
Awesome!
